src/services/git-service.ts

```diff
@@ -0,0 +1,51 @@
+import * as cp from 'node:child_process'
+import * as path from 'node:path'
+import * as fs from 'node:fs'
+
+import { CLIError } from '@oclif/errors'
+
+import debugFactory from './debug-factory-service'
+import FSService, { ServiceParams } from './fs-service'
+
+export class GitService {
+  private static debug = debugFactory(GitService)
+  private readonly options: ServiceParams
+  private readonly fsService: FSService
+
+  constructor(options: ServiceParams) {
```

Comment:
Again, use straight arguments and don't wrap in `ServiceParams` object. I don't see much value in it (also name is too generic)

Reply:
changed `ServiceParams` argument to `bundleName` and `parentDirectory` arguments